Makefile

```diff
@@ -21,27 +21,35 @@ UNITDIR              = unit
 UNIT_TESTSDIR        = $(TESTS_DIR)/$(UNITDIR)
 EXAMPLES_DIR         = examples
 
-SRC := $(shell find $(SRCDIR) -name "*.c")
+SRC := $(wildcard $(SRCDIR)/*.c $(SRCDIR)/platform*/*.c)
```

Comment:
This works for now, but 'assumes' that there is only one sub-dir under `src/` named `platform_<_something_>`.

This creates a small dependency in future if ever we introduce a different sub-dir under `src/`. 

There are some more evolved ways of writing our own recursive `wildcard` function in `Make` lingo, but I found that needlessly complicated to introduce at this time. (Make's `wildcard ` function does not natively do a recursive search thru sub-dirs.)

Reply:
@rtjohnso suggests that we should fix this for good w/o relying on this dependency that `platform*` is the only sub-dir seen under `src/`.

I think you meant this stackoverflow [Recursive wildcards in make](https://stackoverflow.com/questions/2483182/recursive-wildcards-in-gnu-make/18258352#18258352) post, right, which shows how to write a recursive `wildcard` function in make?